ampcrowd/basecrowd/models.py

```diff
@@ -28,6 +28,13 @@ def __unicode__(self):
     class Meta:
         abstract = True
 
+    @staticmethod
+    def to_json(task_group):
```

Comment:
these methods aren't actually outputting JSON. Either have the method return `json.dumps(return_dict)`, or name it `to_dict`().